download_and_create_reference_datasets/v02/hail_scripts/write_clinvar_ht.py

```diff
@@ -1,7 +1,7 @@
 import tempfile
 
 import hail as hl
-from hail_scripts.reference_data.clinvar import (
+from v03_pipeline.lib.reference_data.clinvar import (
```

Comment:
Some decision context: I opted to _move_ the reference data combiner functions to `v03_pipeline` because any use of them is non-automated, and clinvar (at least) can run locally.